apps/api/src/app/events/e2e/bridge-sync.e2e.ts

```diff
@@ -325,12 +325,12 @@ describe('Bridge Sync - /bridge/sync (POST)', async () => {
       {
         preferences: {
           workflow: {
-            defaultValue: false,
+            enabled: false,
```

Comment:
`defaultValue` was uncomfortable, especially when it comes to the Subscriber making changes to the `defaultValue` - it's better renamed to `enabled` to dictate the actual outcome, rather than a relation to the resources modifying it.